sdk/python/telemetry_tests/test_telemetry.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+from feast.client import Client
+from feast.entity import Entity
+from feast.value_type import ValueType
+from google.cloud import bigquery
+import os
+import pytest
+from time import sleep
+
+def test_telemetry_on():
+    os.environ["FEAST_IS_TELEMETRY_TEST"] = 'True'
+    test_client = Client(serving_url=None, core_url=None)
+    test_client.set_project("project1")
+    entity = Entity(
+        name="driver_car_id",
+        description="Car driver id",
+        value_type=ValueType.STRING,
+        labels={"team": "matchmaking"},
+    )
+
+    timestamp = datetime.utcnow()
+    try:
+        test_client.apply(entity)
+    except Exception:
+        pass
+
+    sleep(30)
+    bq_client = bigquery.Client()
+    query = f"select * from `kf-feast.feast_telemetry.cloudfunctions_googleapis_com_cloud_functions_*` where JSON_EXTRACT(textPayload, '$.is_test')='\"True\"' and JSON_EXTRACT(textPayload, '$.timestamp')>'\"{timestamp.isoformat()}\"'"
```

Comment:
Can we add a time range filter here please? This query can become very expensive if we have a lot of data. We should also ensure that there is day partitioning on the BQ table I think.